### PR TITLE
Add v1.11.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
**Todos:**

- [x] Add list of contributors

### CODE REVIEW

File: posts/2017-04-12-serverless-v1.11.0.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Separate Package and Deploy command, CloudWatch Log event support with Serverless v1.11
```
Comment (DavidWells) [inline posts/2017-04-12-serverless-v1.11.0.md]: We need catchier titles focusing on a main benefit.

Suggestion: "New Release: Trigger functions with cloudwatch events in Serverless v1.11"

cc @Stefmonge what do you think?

Comment (DavidWells) [inline posts/2017-04-12-serverless-v1.11.0.md]: or "New in Serverless v.1.11: Trigger functions with via cloudwatch infrastructure events"

Comment (Stefmonge) [inline posts/2017-04-12-serverless-v1.11.0.md]: @DavidWells agreed. Great suggestions!

Comment (Stefmonge) [inline posts/2017-04-12-serverless-v1.11.0.md]: @pmuens do you prefer one of the formats David mentioned over the other? We choose one and stick with one and be consistent.

**Comment by worldsoup** [comment]
idea: for big new features (like triggering functions from cloudwatch events) we create a simple example project and link to it in the blog/newsletter so it's easy for users to jump in and easily use the new feature (also gives them some context on how it can be used) 
@pmuens @eahefnawy thoughts?

File: posts/2017-04-12-serverless-v1.11.0.md (hunk 1 above)
Comment (pmuens) [inline posts/2017-04-12-serverless-v1.11.0.md]: Thanks @DavidWells and @Stefmonge yes, I agree. Will update it.

**Comment by pmuens** [comment]
@worldsoup 

> idea: for big new features (like triggering functions from cloudwatch events) we create a simple example project and link to it in the blog/newsletter so it's easy for users to jump in and easily use the new feature (also gives them some context on how it can be used)
@pmuens @eahefnawy thoughts?

Yes, that's a good idea. We've used code snippets in the past which showed how this could look like. But yes, a more interactive example could be helpful. Best would be to combine all the new features which are included in the release into one example since doing an example for only one feature is lots of work.

**Comment by worldsoup** [comment]
ya I think it could help build up our examples repo and also entice users have stopped using the framework to jump back in.

File: posts/2017-04-12-serverless-v1.11.0.md (hunk 1 above)
Comment (pmuens) [inline posts/2017-04-12-serverless-v1.11.0.md]: Just thought about this problem again. The issue here is that both topics are really interesting and release highlights.

The `package` and `deploy` separation was a super high priority change requested by a bunch of different enterprise users to make CI / CD better.

The `cloudwatchLog` event source on the other hand is super interesting for day-to-day Serverless users 🤔

**Pull request merged**